src/components/Widgets/Widget.tsx

```diff
@@ -225,7 +227,11 @@ export function Widget({
     >
       {isMultisigSigner && <MultisigWalletHeaderAlert />}
       <ClientOnly fallback={<WidgetSkeleton config={config} />}>
-        <LiFiWidget integrator={config.integrator} config={config} />
+        <LiFiWidget
+          integrator={config.integrator}
+          config={config}
+          feeConfig={{ name: 'FlexibleFee', fee: 2, _vcComponent: FlexibleFee }}
```

Comment:
Name here should be the name of the project and it will be shown in the transaction details. Also, the fee parameter range is from 0 to 1 where 1 is 100% 🙂